### ISSUE TITLE
Portals Disappear  and need to visit each one to show on map

### ISSUE THREAD

**Issue by NanakoMiko** [blaxxun-boop/TargetPortal#11]
Hi

I run a Dedicated server and all my friends, can't see all the portals even when its public or in a group (Group mod) I tested this and had to take a long trip just to see if I access a portal a friend placed I was able to see all his portals placed, public portals

Is this how it works, you need to visit each portal once before you can see them?

Didn't see a discord so this is the only way I can contact you.

I removed the mod for the time being as it is not working well for us. Hopefully it might be a bug or something.

**Comment by blaxxun-boop** [comment]
No, that's not how it works and that's actually caused by another mod. I cannot remember the name, though.

**Comment by NanakoMiko** [comment]
Can you please help in providing some assistance I am struggling to get the mod that breaking yours, i have screenshots of my mods

i took over an hour to figure out which mods it is with my friend but we can't find the problematic mod.

Thanks

**Comment by evilHarry64** [comment]
I had also to remove the mod from the Server because of this problem portals not showing up on the server...
if another mod conflicts with this which mod is it?
I have on the Server:

ItemDrawers
EpicLoot
EpicLoot_Deutsch
EpicLoot_ItemDrawers_Visual_Patch
ExtendedItemDataFramework
MultiCraft
MassFarming

I really would love to have this mod on the server...

**Comment by NanakoMiko** [comment]
> I had also to remove the mod from the Server because of this problem portals not showing up on the server... if another mod conflicts with this which mod is it? I have on the Server:
>
> ItemDrawers EpicLoot EpicLoot_Deutsch EpicLoot_ItemDrawers_Visual_Patch ExtendedItemDataFramework MultiCraft MassFarming
>
> I really would love to have this mod on the server...

You don't have that many mods on this server, I would suggest disabling EpicLoot and all that is relevant to it to test.
However, you will most likely lose items that are related to Epicloot, make a backup of your server before you attempt fining the culprit mod.

If you are able to disable all mods and start with the target portal and enable each mod to see which one breaks it

Hope this helps.

**Comment by evilHarry64** [comment]
It seems to work if I try it in a Local Game... all portals show up instantly.
the server thing I would need to try when I am not busy.

question... if I try this on the Server that I manage and I find the problematic mod, would you consider looking into it to fix this or would you not... because my people would not like to loose any of the mods I have currently installed.

it would be a shame because your mod is such an elegant solution. exactly what I was looking for.

**Comment by evilHarry64** [comment]
> It seems to work if I try it in a Local Game... all portals show up instantly. the server thing I would need to try when I am not busy.
> 
> question... if I try this on the Server that I manage and I find the problematic mod, would you consider looking into it to fix this or would you not... because my people would not like to loose any of the mods I have currently installed.
> 
> it would be a shame because your mod is such an elegant solution. exactly what I was looking for.

o.. I am dumb...
All working now.

I didn´t notice it but this server I am managing has 2 profiles.
one standard without mods (that was active apparently)
and one with Valheim Plus preinstalled where all my mods where copied to.

I did not realize that the one without mods was active because every other mod appeared to work compleatly fine.
the only other one that had a little problem was the EpicLoot mod.
I could not complete any quests the trader offered me.
I looked into this problem and found a post of someone having the same issue and it turned out none of his mods loaded on the server and this was the only code that needs to be executed on the server to work.

I write this just in case anyone has the same problem. ^__^
